en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+# Concepts
+
+Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
+ 
+## Integration
+
+Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
+
+
+## Integration Template
+
+An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+
+
+## Trigger
+
+A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+
+Choreo supports the following trigger types:
+
+ - **API**: Allows you to expose an application as an API endpoint. 
+ - **Manual**: Allows you to trigger an application manually.
+ - **Schedule**: Allows to trigger an application based on a schedule that you set. You can specify the schedule in the form of a cron expression.
+   <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p> A cron expression represents details of a schedule. It is a string that contains subfields separated by whitespaces. Each special character (*) represents seconds, minutes, hours, date, month, day, and year respectively.</p>
+     </div>
+     </html> 
+ - **Calendar**: Allows you to trigger an application based on events in a Google Calendar.
+ - **GitHub**: Allows you to trigger an application based on GitHub events and actions. The supported GitHub events and associated actions that can trigger a Choreo application are as follows:
+
+   | Event                       | Action                 | Application Trigger                      |
+   |-----------------------------|------------------------|------------------------------------------|
+   | issue_comment               | created                | Add a comment to an issue                |
+   |                             | edited                 | Edit a comment on an issue               |
+   |                             | deleted                | Delete a comment on an issue             |
+   | issues                      | assigned               | Assign an issue to a user                |
+   |                             | unassigned             | Unassign an issue from a user            |
+   |                             | labeled                | Add a label to an issue                  |
+   |                             | unlabeled              | Remove a label from an issue             |
+   |                             | opened                 | Open an issue                            |
+   |                             | edited                 | edit an issue                            |
```

Comment:
"edit an issue" starts with a simple "e" here. It is inconsistent with the rest of the rows.

Reply:
Updated the capitalization.